rollbar/test/__init__.py

```diff
@@ -1,9 +1,12 @@
-import unittest2
+try:
+    import unittest2 as unittest
```

Comment:
Thanks for your change. I don't think we even need to fallback on unittest2 as Python 2+ has been EOLed. Just move ahead with replacing it to unittest.

Reply:
Well, it doesn’t save me. We have `rollbar` in Leap 15.* so we still build python2- packages. I don’t want to knowingly break python2 compatibility if I can help it. Do you know `rollbar` functionality available only in Python 3.* or unittest2 respectively?

Reply:
@mcepl, Thank you for your feedback. We are definitely not removing the support for python2 yet. If we can replace unittest2 with unittest without breaking existing tests and builds that will be great.